### ISSUE TITLE
fuse_ab for RepOpt

### ISSUE THREAD

**Issue by QiqLiang** [meituan/YOLOv6#970]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

Hi, i got the following error when finetuning my yolov6n model:
AttributeError: 'ConfigDict' object has no attribute 'anchors_init'
It is P5 model and its training mode is repopt. i want to use fuse_ab for finetuning(but my pretrained model didn't set fuse_ab) and got this error. Why does it happen and how could i solve it?

**Comment by Chilicyy** [comment]
Hi @QiqLiang If you want to use fuse_ab, please make sure there's the initiation of anchor in the config file, otherwise you can add the line `anchors_init=[[10,13, 19,19, 33,23], [30,61, 59,59, 59,119], [116,90, 185,185, 373,326]],` , referring to https://github.com/meituan/YOLOv6/blob/e9656c307ae62032f40b39c7a7a5ccc31c2f0242/configs/yolov6n_finetune.py#L25C9-L27C51.

**Comment by QiqLiang** [comment]
> Hi @QiqLiang If you want to use fuse_ab, please make sure there's the initiation of anchor in the config file, otherwise you can add the line `anchors_init=[[10,13, 19,19, 33,23], [30,61, 59,59, 59,119], [116,90, 185,185, 373,326]],` , referring to https://github.com/meituan/YOLOv6/blob/e9656c307ae62032f40b39c7a7a5ccc31c2f0242/configs/yolov6n_finetune.py#L25C9-L27C51.

yes, i have already set this. Is it because the scale.pt and pretrained file don' have the corresponding fuse_ab branch so i can't finetune based on them?

**Comment by childishPilot** [comment]
> > Hi @QiqLiang If you want to use fuse_ab, please make sure there's the initiation of anchor in the config file, otherwise you can add the line `anchors_init=[[10,13, 19,19, 33,23], [30,61, 59,59, 59,119], [116,90, 185,185, 373,326]],` , referring to https://github.com/meituan/YOLOv6/blob/e9656c307ae62032f40b39c7a7a5ccc31c2f0242/configs/yolov6n_finetune.py#L25C9-L27C51.
> 
> yes, i have already set this. Is it because the scale.pt and pretrained file don' have the corresponding fuse_ab branch so i can't finetune based on them?

Hi, I have a same issue how could you solve it?
